.github/workflows/inventory.yml

```diff
@@ -0,0 +1,82 @@
+name: Generate Inventory Report
+
+on:
+  workflow_dispatch:
+    inputs:
+      last-modified-date:
+        description: >
+          Last modified date of inventory objects to select, either formated as
+          YYYY-MM-DD (e.g., 2025-06-10) or YYYYDDD (e.g., 2025161, equivalent to
+          2025-06-10), where DDD is day of year.  (Default: 2 days ago)
+        required: false
+        type: string
+      products:
+        description: >
+          Comma-separated list of products to produce reports for.  (Default:
+          L30, L30_VI, S30, S30_VI)
+        required: false
+        type: string
+      lambda-function-name:
+        description: Name or ARN of the report-generator Lambda function
+        required: false
+        type: string
+        # TODO Find reliable means of looking this up in a job step
+        default: hls-lpdaac-reconciliation-InventoryReportHandler89-HlYwwIOfV3yw
```

Comment:
We'd have to define the stack name as a GH environment variable but I could imagine something like,

1. Define the stack name (or maybe just template it from the environment name)
2. Add the function as a stack output, something like `ReportGenerationLambda`
3. Fetch it using the AWS CLI like, `aws cloudformation describe-stacks --stack-name <stack-name> --query "Stacks[0].Outputs[?OutputKey=='<output-key>'].OutputValue" --output text` where `<stack-name>` and `<output-key>` are variables

I could also see us doing this in Python code if we have `boto3`

Reply:
Ah, `describe-stacks`. That's the command I was looking for. Let me see what I can wrangle.